### PR TITLE
Added video tutorial for Linked List Basic Syntax understanding

### PR DESCRIPTION
**Pull request by adityarajsahoo**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes # <issue_number_here> 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>
Video

### CODE REVIEW

File: C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Adding video tutorial drive link for understanding linked list basic syntax and code.
```
Comment (Suvraneel) [inline C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md:1]: Observe & do the same
https://github.com/girlscript/winter-of-contributing/pull/3407/files

**Review by Suvraneel — CHANGES_REQUESTED** [review]

File: C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md (hunk 1 above)
Comment (adityarajsahoo) [reply C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md]: I made the required changes. Can you check bhaiya?

Comment (adityarajsahoo) [reply C_CPP/Data Structures/Linked List/Linked_List_Basic_code_Video.md]: But I am facing the issue of not getting thumbnail of image. So I tried through google drive but it still does not work.

**Comment by adityarajsahoo** [comment]
@Suvraneel bhaiya Can you tell how to add image? I tried but it's not displaying here

**Comment by Suvraneel** [comment]
@adityarajsahoo  Copy any image (serves as thumbnail for video) and paste it here (Ctrl + V)

**Comment by adityarajsahoo** [comment]
Thanks bhaiya. I made the required changes.

**Comment by adityarajsahoo** [comment]
@Suvraneel I have made the required changes. Can you review?

**Comment by Suvraneel** [comment]
@adityarajsahoo  Almosttttt... All links are in parenthesis
I've made a minor change in the bracket structure... You can do the same next time

**Comment by Suvraneel** [comment]
@adityarajsahoo  LGTM
Good work ❤️ 🚀

**Comment by adityarajsahoo** [comment]
Thanks a lot bhaiya!

**Pull request merged**